package.json

```diff
@@ -8,14 +8,15 @@
     "json",
     "parser"
   ],
-  "version": "2.0.2",
+  "version": "2.1.0",
```

Comment:
updating the version is handled by the release. and it would be a major release since the PR would drop support for older node versions